templates/changeVobLocation/gamesave.d

```diff
@@ -28,6 +28,6 @@ func int G1CP_{ISSUE_NUM_PAD}_{SHORTNAME}Revert() {
         return FALSE;
     };
 
-    return G1CP_ChangeVobLocation(G1CP_{ISSUE_NUM_PAD}_{SHORTNAME}_CorrectedPos, 
-        G1CP_{ISSUE_NUM_PAD}_{SHORTNAME}_OriginalPos, Hlp_Is_WhateverYouSeek);
-};
\ No newline at end of file
+    return G1CP_ChangeVobLocation(_@(G1CP_{ISSUE_NUM_PAD}_{SHORTNAME}_CorrectedPos),
+        _@(G1CP_{ISSUE_NUM_PAD}_{SHORTNAME}_OriginalPos), Hlp_Is_Todo);
```

Comment:
Another name for the placeholder `Hlp_Is_Todo` could be `callback` to make it clearer what it is. (I was confused what "Hlp_Is_Todo" means). A comment with "TODO" next to it (see other review comment) will make it quite convenient and clear.